R/a11y_panel.R

```diff
@@ -107,6 +107,7 @@ a11y_panel <- function(
     stop("Error: If publication_slug is provided, then so should publication_name.")
   }
   shiny::tags$div(
+    style = "margin-top: 50px;",
```

Comment:
Let's add bottom margin of the same size as well whilst we're setting this, so that there's some decent separation when people use the footer function.